SYCL/ESIMD/esimd_test_utils.hpp

```diff
@@ -6,6 +6,8 @@
 //
 //===----------------------------------------------------------------------===//
 
+#pragma once
+
 #include <CL/sycl.hpp>
```

Comment:
```suggestion
#include <sycl/sycl.hpp>
```
While you a here :) Great catch with pragma BTW

Reply:
This comment has been applied [here](https://github.com/intel/llvm-test-suite/pull/789/commits/4d4d19242eb873b792da11f9a118dd5ef6e9c6cd#diff-c73d28d272bf069fdc585bd82fc0fcbcba32f9b98a256e21781cfedc5080d0e5R11)